### DISCUSSION TITLE
Pb Material

### DISCUSSION THREAD

**Discussion by kojiro18** [arkypita/LaserGRBL#1092] [Q&A]
Hello, I translated the original lasergrbl material base into French. I would like to add new materials and share them with my groups using a file how can I do I am blocked thank you

**Comment by arkypita** [comment]
Material DB is not translatable.
I mean, yes you can translate but is not supported with multiple language so I cannot deploy it with new versions.

However you can find it in C:\Users\YourName\AppData\Roaming\LaserGRBL 
The file is called UserMaterial.psh and you can share this file manually

**Comment by ZASto** [comment]
So, how is Material DB used in your program?
Haven't seen any place a possibility to use data from the database.

**Comment by kojiro18** [comment]
Comment ajouter manuellement dans le fichier

**Comment by StuartB4** [comment]
Looks like you need to have an Ortur machine to use the DB. Theres a little book icon in the top right corner of Target image settings, but for me it's greyed out so I can't change anything in there.
![yuyuyuyuyuyu](https://user-images.githubusercontent.com/16980026/93317012-d37d8e00-f804-11ea-82a5-06a1db8fb75f.jpg)
